src/zcash/Note.cpp

```diff
@@ -69,14 +70,14 @@ std::optional<uint256> SaplingNote::cmu() const {
     // We consider ZIP 216 active all of the time because blocks prior to NU5
     // activation (on mainnet and testnet) did not contain Sapling transactions
     // that violated its canonicity rule.
-    if (!librustzcash_sapling_compute_cmu(
-            d.data(),
-            pk_d.begin(),
+    try {
+        result = uint256::FromRawBytes(sapling::spec::compute_cmu(
+            d,
+            pk_d.GetRawBytes(),
             value(),
-            rcm_tmp.begin(),
-            result.begin()
-        ))
-    {
+            rcm_tmp.GetRawBytes()
+        ));
+    } catch (rust::Error) {
         return std::nullopt;
```

Comment:
Should these errors be logged at all?

Reply:
We probably could now, but I don't know how noisy that would get. The old code did not log these errors, and the new code maintains that approach. We should probably look at the cross-FFI errors and logging as a whole.